app.py

```diff
@@ -23,6 +23,10 @@
 app.secret_key = os.environ['SECRET_KEY']
 app.wtf_csrf_secret_key = os.environ['WTF_CSRF_SECRET_KEY']
 app.url_map.strict_slashes = False
+app.sentry_public_dsn = os.getenv(
```

Comment:
can you rebase on master so that you can modify the already existing var?

Reply:
I rebased, but nothing changed!?